how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,540 @@
+    "\n",
+    "**Note**: To log or retrieve metadata from Neptune, you need the `project name` and the `api_token`.\n",
+    "\n",
+    "To make this example easy to follow, we have created a public project **'common/showroom'** and a shared user **'neptuner'** with the API token **'ANONYMOUS'**. As you will see in the code cell below.\n",
+    "\n",
+    "**(Optional)** If you want to log to your own project you have to have or create [Neptune account](https://app.neptune.ai/register/) and [project](https://docs.neptune.ai/getting-started/installation#setting-the-project-name).\n",
+    "Then you can pass [project](https://docs.neptune.ai/getting-started/installation#setting-the-project-name) and [api_token](https://docs.neptune.ai/getting-started/installation#authentication-neptune-api-token) arguments to the `init()` method.\n",
+    "\n",
+    "`run = neptune.init(api_token='<YOUR_API_TOKEN>', project='<YOUR_WORKSPACE/YOUR_PROJECT>')` \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Bod3d1otHtn7"
+   },
+   "outputs": [],
+   "source": [
+    "import neptune.new as neptune\n",
+    "\n",
+    "# Fetch project\n",
+    "project = neptune.get_project(name=\"common/showroom\", api_token=\"ANONYMOUS\")\n",
+    "\n",
+    "# Fetch only inactive runs with tag \"showcase-run\"\n",
+    "runs_table_df = project.fetch_runs_table(state=\"idle\", tag=[\"showcase-run\"]).to_pandas()\n",
+    "\n",
+    "# Extract the last failed run's id\n",
+    "failed_run_id = runs_table_df[runs_table_df[\"sys/failed\"] == True][\"sys/id\"].values[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 35
+    },
+    "id": "eEFG0Jn8PNNg",
+    "outputId": "6b751bfa-a254-4995-f1ac-9e48953695f5"
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Failed_run_id = \", failed_run_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "kfqmkNLB1SRu"
+   },
+   "source": [
+    "## Step 2: Resume failed run\n",
+    "Use the `neptune.init()` method to:\n",
+    "* Re-open a run using the ID you got from the previous step \n",
+    "* Re-open it in the `read-only` mode\n",
+    "\n",
+    "You use the `read-only` mode so the metadata previously logged to the run is not accidentally changed. Also, you can re-open a run as many times as needed.\n",
+    "\n",
+    "**(Optional)** If you already have an existing [Neptune account](https://app.neptune.ai/register/) you can pass your credentials to **[project](https://docs.neptune.ai/getting-started/installation#setting-the-project-name)** and **[api_token](https://docs.neptune.ai/getting-started/installation#authentication-neptune-api-token)** arguments of neptune.init()\n",
+    "\n",
+    "`run = neptune.init(api_token='<YOUR_API_TOKEN>', project='<YOUR_WORKSPACE/YOUR_PROJECT>')` "
```

Comment:
Format code snippets within comments with black, and follow latest styleguide (no `<`, `>`)